tasks/spec_runner.rake

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+require 'rspec/core'
+require 'test_queue'
+require 'test_queue/runner/rspec'
+
+module RuboCop
+  # Helper class for running the specs for RuboCop.
```

Comment:
I think you can put some of the rationale from your commit messages in the class description here.